Lib/test/test_ftplib.py

```diff
@@ -447,7 +450,7 @@ def test_exceptions(self):
 
     def test_all_errors(self):
         exceptions = (ftplib.error_reply, ftplib.error_temp, ftplib.error_perm,
-                      ftplib.error_proto, ftplib.Error, IOError, EOFError)
+                      ftplib.error_proto, ftplib.Error, OSError, EOFError)
```

Comment:
What is the rationale for this change? It seems specific to Python 3 where all OS-like exceptions have be replaced by a single OSError. On Python 2.7, you really want IOError. Please revert this change, to repair failing tests.